Pack-Smart/pack-smart-fe
# Phase4 cleanup

## What's this PR do?
- Adds `Start New List` button to header and Hamburger menu
- Add turquoise background to browser tab image
- Makes `StaticCategory` + button toggle between showing and hiding items
- Removes background color on pencil and plus icons
- Adjust sizes of checkboxes on smaller screens
- Consolidates `PackingList` title, duration and destination
- Adds + and - buttons on either sides of the quantity input
- Makes `Hamburger` return dynamic and puts data in separate file

## Where should the reviewer start?
- `StaticCategory.js`, `PackingList.js`, `HamburgerMenu.js` and `StaticItem.js`

## How should this be manually tested?
- View a packing list and adjust the quantities and open and close the list using the +/- buttons
- View each page view and on multiple screen sizes to ensure styles are consistent and look good!
- View the browser tab image (it should now be turquoise)

## What are the relevant tickets?
- Closes #68 
- Closes #77 
- Closes #81

## Screenshots/GIFs
- <img width="135" alt="Screen Shot 2021-03-01 at 8 45 11 AM" src="https://user-images.githubusercontent.com/58453566/109524597-f4ae0000-7a6d-11eb-849a-fdb8f84c718f.png">
- <img width="852" alt="Screen Shot 2021-03-01 at 8 46 02 AM" src="https://user-images.githubusercontent.com/58453566/109524666-068fa300-7a6e-11eb-85bd-ccb633958e02.png">
- <img width="871" alt="Screen Shot 2021-03-01 at 9 02 34 AM" src="https://user-images.githubusercontent.com/58453566/109524719-10b1a180-7a6e-11eb-8a2e-c93fd96f2645.png">

## Questions:
- We had talked about the landing page styles (not being merged on this branch), should we do that on another branch?

## Did you update the README?
- NO! (such a needy question.... lol jk)
